trax/data/bert_inputs.py

```diff
@@ -105,18 +110,14 @@ def tfds_inputs(
   )
 
 
-@gin.configurable()
-def bert_tokenizer(vocab_path=None):
-  """Constructs a BERT tokenizer."""
-  # This import is from https://github.com/google-research/bert which is not
-  # listed as a dependency in trax.
-  # TODO(piotrekp1): using SubwordTextEncoder instead after fixing the
-  # differences
-  from bert.tokenization.bert_tokenization import FullTokenizer  # pylint: disable=g-import-not-at-top
+def bert_tokenizer(vocab_path=None, model_name=None, do_lower_case=True):
```

Comment:
Hopefully this function can disappear in favor of trax.data.tokenize ; Instead of a do_lower_case argument I'd make a separate vocab type though, so I guess 'bert' and 'bert_lowercase'.

Reply:
as you suggested it is gone in new version, nevertheless I created BertEncoder class in data.text_encoder.py and its instance is created by trax.data.tokenize